vault/identity_store_aliases.go

```diff
@@ -206,19 +206,10 @@ func (i *IdentityStore) handleAliasUpdateCommon(req *logical.Request, d *framewo
 	}
 
 	// Get entity id
-	canonicalID := d.Get("entity_id").(string)
+	canonicalID := d.Get("canonical_id").(string)
```

Comment:
Can you also put a deprecation notice for `entity_id`'s pathHelp?

Reply:
Done.